app/src/main/java/com/farmerbb/secondscreen/receiver/TaskerConditionReceiver.java

```diff
@@ -28,18 +28,24 @@
 // Receiver run by Tasker periodically to check the state of currently active profiles, whenever
 // a SecondScreen state is included as a condition in a Tasker profile.
 public final class TaskerConditionReceiver extends BroadcastReceiver {
+    Bundle lastbundle = null;
+
+
     @SuppressWarnings("deprecation")
     @Override
     public void onReceive(Context context, Intent intent) {
+        if (lastbundle.equals(intent.getBundleExtra(com.twofortyfouram.locale.api.Intent.EXTRA_BUNDLE))) {
+            return;
+        }
+        updateValues(intent);
         if(U.isExternalAccessDisabled(context)) return;
```

Comment:
This line should be moved above the four lines that are newly added.  If the external access pref is disabled then the receiver should not run anything, period.